prac_05/word_occurrences.py

```diff
@@ -0,0 +1,18 @@
+TEXT_STRING = "this is a collection of words of nice words this is a fun thing it is"
+
```

Comment:
Could you make this program work for any string given by a user?

Reply:
Add the capability, thank you.